modules/fetching-vulnerability-definitions.adoc

```diff
@@ -5,8 +5,9 @@
 [id="fetching-vulnerability-definitions_{context}"]
 = Fetching vulnerability definitions
 
+//verify this info is still accurate
 In online mode, Central fetches the vulnerability definitions every 5 minutes from a single feed.
-This feed combines vulnerability definitions from upstream sources that include multiple Linux distributions and the National Vulnerability Database, and it refreshes every hour.
+This feed combines vulnerability definitions from upstream sources that include multiple Linux distributions, the National Vulnerability Database, and other sources, and it refreshes every hour.
 
 * The address of the feed is `\https://definitions.stackrox.io`.
```

Comment:
is there supposed to be a leading `\`?

Reply:
Yes, URLs are automatically transformed when publishing, so it creates a clickable link that goes to an XML page (not the best user experience). The `\` character prevents that.

Reply:
![Screenshot 2024-03-14 at 12 24 36 PM](https://github.com/openshift/openshift-docs/assets/96192119/aca9bd70-edd5-4ee9-b012-1959d5af5281)